Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -95,7 +95,7 @@ screen initialState =
         ( \push -> do
           _ <- pure $ JB.checkAndAskNotificationPermission false
           _ <- pure $ printLog "initial State" initialState
-          _ <- if initialState.data.config.enableMockLocation then JB.isMockLocation push IsMockLocation else pure unit
+          -- _ <- if initialState.data.config.enableMockLocation then JB.isMockLocation push IsMockLocation else pure unit
```

Comment:
please add a comment as well along with the commented code, what is this commented code about